web/src/widgets/ApkWidget/index.jsx

```diff
@@ -0,0 +1,112 @@
+
+function ApkWidget() {
+    const toast = useToast()
+    const [selectedFile, setSelectedFile] = useState(null);
+    const [putAkpUploadFile, { isLoading: isPuttingApkUpload, error: errorPuttingFile }] = usePutApkUploadFileMutation()
+    const { trigger: getWebConfigurations, data: responseData, error, isLoading } = useAxios({ mainUrl: `${BASE_API_URI}/get-apk`, useAuthorisation: false });
+    const [webConfigurations, setWebConfigurations] = useState(null);
+
+
+    // HANDLE FILE CASE
+    const handleFileChange = (event) => {
+        const file = event.target.files[0];
+        setSelectedFile(file);
+
+    };
+
+    const handleFormSubmit = async (e) => {
+        e.preventDefault();
+    
+        if (!selectedFile) {
+            // Handle case when no file is selected
+            toast({
+                position: 'top-center',
+                title: 'Error',
+                description: 'Please select a picture to upload.',
+                status: 'error',
+                duration: 2000,
+                isClosable: true,
+            });
+            return;
+        }
+    
+        const formData = new FormData();
+        formData.append('file', selectedFile);
+    
+        try {
+            const response = await putAkpUploadFile(formData).unwrap();
+            const message = response["message"];
+            const errormessage = response["error_message"];
+    
+            if (message !== undefined && message !== null) {
+                toast({
+                    position: 'top-center',
+                    title: 'Success',
+                    description: message,
+                    status: 'success',
+                    duration: 2000,
+                    isClosable: true,
+                });
+                setSelectedFile(null)
+            } else if (errormessage !== undefined && errormessage !== null) {
+                toast({
+                    position: 'top-center',
+                    title: 'An error occurred',
+                    description: errormessage,
+                    status: 'error',
+                    duration: 2000,
+                    isClosable: true,
+                });
+            }
+        } catch (error) {
+            console.error('Error uploading picture:', error);
+        }
+    };
+    
+    useEffect(() => {
+        getWebConfigurations();
+    }, []);
+
+    useEffect(() => {
+        if (responseData?.configurations) {
+            setWebConfigurations(responseData.configurations);
+        }
+    }, [responseData]);
+    
+    return (
+        <Fragment>
+            <form className="row bio-data p-3" onSubmit={handleFormSubmit}>
+               
+            {webConfigurations?.version
+                && <p><a className='badge bg-primary'>App Current Version: {webConfigurations.version}</a></p>
+            }
+                <div className="mb-3 col-md-12">
+                    <label htmlFor="formFile" className="form-label">Upload apk file</label>
```

Comment:
Also show the download link on the setup page as shown here.
<img width="1073" alt="image" src="https://github.com/dodziraynard/y-check-ug/assets/45526474/6090abdf-76d3-4378-9c65-b177fe29943d">

Reply:
done
<img width="1036" alt="image" src="https://github.com/dodziraynard/y-check-ug/assets/81252456/08348208-819c-448d-a3d5-47dd31a19ea1">